src/main/java/io/appium/java_client/ios/IOSDriver.java

```diff
@@ -220,4 +224,50 @@ public T findElementByIosUIAutomation(String using)
     public void lockDevice(int seconds) {
         CommandExecutionHelper.execute(this, lockDeviceCommand(seconds));
     }
+
+    @Override public TargetLocator switchTo() {
+        return new InnerTargetLocator();
+    }
+
+    private class InnerTargetLocator extends RemoteTargetLocator {
+        @Override public Alert alert() {
+            return new IOSAlert(super.alert());
+        }
+    }
+
+
+    class IOSAlert implements Alert {
+
+        private final Alert alert;
+
+        IOSAlert(Alert alert) {
+            this.alert = alert;
+        }
+
+        @Override public void dismiss() {
+            execute(DriverCommand.DISMISS_ALERT);
```

Comment:
@TikhomirovSergey ```alert.dismiss()``` cannot be used as ```RemoteWebDriver``` uses ```RemoteWebDriver.this.execute("dimissAlertW3C")``` i.e W3C API which we didn't adopt yet. It is the same for ```alert.accept()``` as well